components/LoadingScreen.tsx

```diff
@@ -4,9 +4,9 @@ import { ActivityIndicator } from 'react-native-paper'
 import { useThemeColor } from './Themed'
 
 interface Props {
-  loading: boolean
+  loading?: boolean
```

Comment:
Changed this to optional so that the current tests written for this component pass typechecking. 

Also provided a default value of false to ensure loading is always set to at least something